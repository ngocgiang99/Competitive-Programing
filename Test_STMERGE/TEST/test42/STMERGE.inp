7
1 2
846368152 710364044 
1 6
276532684 972256992 516329608 848280140 288862279 893475197 
7 9
282631212 6936088 223950059 326507007 778655456 195972006 90008962 550609302 151370762 
582908764 602641911 39880200 929075597 343738542 542417489 611994345 601432974 525414227 
153519828 164120590 480823412 369247551 503360005 25315213 569098978 712584870 135012521 
718119354 358643133 930970110 822206405 295954731 88275499 881182716 489417374 213654196 
25015185 439763861 130973882 117294442 235425172 179386366 172959197 227032739 538354200 
792105132 551667211 776706035 882268482 381801812 71104395 171271471 429410681 959415274 
368558817 356108909 209256800 176041637 272727763 42894849 423178205 728563704 766072420 
5 10
469460121 595849947 335022785 78163819 947733885 255969211 316216791 715775878 124942809 613118667 
955571479 942752844 164552994 878501903 520384022 910738400 897614093 499809740 786594985 273346727 
903969167 547469447 877808358 220575559 7745757 824054022 282975010 136667813 730356587 818840140 
744774641 701427685 87355375 862560160 684119080 346501230 781839199 675968645 681519324 96387042 
618833134 189689538 717511011 646366761 478223890 266538587 878728530 724075878 770647168 513344406 
1 8
644882989 665184502 657628921 576634008 768097101 978835265 460858139 13294614 
7 5
877743481 200205216 498445822 457931502 634004200 
442494105 267721529 148187417 183022715 659304615 
975886139 667847619 292630921 548967343 752913786 
610752241 172519335 997479893 595963307 430922472 
905184425 769349502 117694403 349549102 103294116 
184767825 189455863 641737606 867902958 111257649 
35945660 509723639 69116699 688083268 7341767 
5 4
157509614 359097612 648775541 919126707 
771260373 458978497 907476428 829387144 
354778333 482877788 770993615 40792853 
564511533 124036240 965009348 147073838 
453161256 549892179 547721262 714317288 
