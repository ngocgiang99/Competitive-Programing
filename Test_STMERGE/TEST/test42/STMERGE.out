710364044
276532684
151370762
182296618
13294614
35945660
453161256
